tests/jballerina-unit-test/src/test/resources/test-src/bala/test_bala/types/xmlns_bala_test.bal

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import testorg/foo;
+
+xmlns foo:XMLNSA as ns0;
+xmlns foo:XMLNSB as ns1;
+
+function testXMLNSDeclUsingModuleConstant() {
+    string s1 = ns0:foo;
```

Comment:
Is there a specific reason for doing this assignment? If not, we can directly use `ns0:foo` for assertion without assigning it.

Reply:
Fixed